common/models/report.js

```diff
@@ -1,12 +1,84 @@
-var countries = require('country-data').countries;
+var axios = require('axios')
+var qs = require('qs')
+
+var countries = require('country-data').countries
+
+var apiClient = axios.create({
+  // XXX Uncomment this when we deploy it
+  // baseURL: 'https://api.ooni.io/api/',
+  // XXX We use this base URL for testing
+  baseURL: 'http://localhost:3001/api/',
+  timeout: 90000, // Maybe set this lower once performance is boosted
+})
```

Comment:
Is this code executed in context of browser? I assume, it is.
Is there an easy way to take baseURL from an environment or a file? I assume, that's static file served to browser without "build" stage and "sed during deploy" is the "best" way.

Reply:
Perhaps, but I didn't find a quick and easy way to do that, so for the moment i left it like this.

To be honest I don't want to spend too much time making this code super clean and elegant.